lib/helpers/path-diff.js

```diff
@@ -0,0 +1,30 @@
+// @ts-check
+/* eslint-disable no-continue */
+const { basename, normalize } = require('path');
+const isGlob = require('is-glob');
+
+const leadingSlash = (val) => val.startsWith('/') ? val : `/${val}`;
```

Comment:
If we need this type of checks, would it make sense moving this and the two found [here](https://github.com/eik-lib/common/pull/103/files#diff-2a8a60d95f6c8a7f0bc3d7c30c319a2889895c59f7ed77dee91e33cf5a02eb03R24-R33) into a util module?

Reply:
yeah, I likely would. Can take a refactor round and do some cleanup.

Reply:
:+1:

Reply:
Cleanup done